packages/matchbox/src/components/Pager/Next.tsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
 import { ArrowForward } from '@sparkpost/matchbox-icons';
-import { Button } from '../Button';
+import { Button, PolymorphicButton } from '../Button';
 import { ScreenReaderOnly } from '../ScreenReaderOnly';
 
-type NextProps = React.ComponentProps<typeof Button>;
-
-const Next = (props: NextProps): JSX.Element => (
-  <Button size="small" width="600" {...props} ml={props.ml || props.marginRight || 200}>
-    <ArrowForward size={16} />
-    <ScreenReaderOnly>Next</ScreenReaderOnly>
-  </Button>
-);
+const Next = React.forwardRef(function Next(props, userRef) {
+  return (
+    <Button
+      size="small"
+      width="600"
+      {...props}
+      ml={props.ml || props.marginRight || 200}
+      ref={userRef}
+    >
+      <ArrowForward size={16} />
+      <ScreenReaderOnly>Next</ScreenReaderOnly>
+    </Button>
+  );
+}) as PolymorphicButton;
```

Comment:
Need to verify this works

Reply:
Getting `Type '"a"' is not assignable to type '"button"'.`

Reply:
* When testing the `as` prop using `a`